docs/references/cmcspec.mdx

```diff
@@ -0,0 +1,552 @@
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Cycles Minting Canister (CMC) API
+
+<MarkdownChipRow labels={["Reference"]} />
+
+The Cycles Minting Canister (CMC) is a system canister on the Internet Computer responsible for converting ICP tokens into **cycles**, tracking the total cycles minted, and providing configuration and exchange rate data. It supports:
+
+-   Topping up existing canisters.
+-   Creating new canisters.
+-   Minting and depositing minted cycles into cycle ledger-managed accounts.
+-   Fetching exchange rates and subnet configuration.
+
+The CMC is governed solely by the [NNS (Network Nervous System)](https://internetcomputer.org/nns/) and receives configuration updates through proposals.
```

Comment:
"receives configuration updates" - this is true, but not exhaustive.  It also receives new functionality, bug fixes, etc.

Something more general, like "updates" or "updates to functionality and configuration" might be clearer to those who aren't already IC aficionados.